src/test/java/com/salesforce/mirus/config/SourceConfigTest.java

```diff
@@ -34,6 +41,58 @@ public void setUp() {
     mirusSourceConfig = new SourceConfig(properties);
   }
 
+  public static class SimpleTransformation<R extends ConnectRecord<R>>
+      implements Transformation<R> {
+
+    int magicNumber = 0;
+
+    @Override
+    public int hashCode() {
+      final int prime = 31;
+      int result = 1;
+      result = prime * result + this.magicNumber;
+      return result;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+      if (obj instanceof SimpleTransformation) {
+        SimpleTransformation other = (SimpleTransformation) obj;
+        if (this.magicNumber == other.magicNumber) {
+          return true;
+        }
+      }
+      return false;
```

Comment:
Given that this is a single-use test class, can you just condense `hashCode` and `equals` into one- or two-liners for the sake of reducing visual noise?